File: setup/db_setup.js
// Node.JS imports
var sqlite3 = require('sqlite3').verbose();
var fs = require('fs');

// Delete the old DB
if(fs.existsSync('../dbs/users.sql')) {
	fs.unlink('../dbs/users.sql', function(err) {
		if(err) {
			console.log(err);
		} else {
			// Generate the new DB
			gen_db();
		};
	});
} else {
	gen_db();
};

// Create the DB
function gen_db() {
	var db = new sqlite3.Database('../dbs/users.sql');

	// Create the table
	db.run("CREATE TABLE main.users (UUID text, username text, password text)");
};

File: setup/db_setup.py
#!/usr/bin/python3

# Python Imports
import sqlite3
import os

# Delete the old database
os.remove('./dbs/users.sql');

# Define the Database
db = sqlite3.connect('./dbs/users.sql');
print 'Connected to DB';

# Create the tables
db.execute('CREATE TABLE main.users (UUID text, username text, password text, rand text)');
print 'Database Created';
